utils/postprocess/elasticity_postprocessing.py

```diff
@@ -235,24 +260,328 @@ def solutionFieldOnMeshToVtk3D(geom,
+            
+            if analytical_solution:
+                u_analytical, v_analytical, sigma_xx_analytical, sigma_yy_analytical, sigma_xy_analytical = analytical_solution(X, t=current_time)
+                
+                combined_disp_true = tuple(np.vstack((u_analytical.flatten().tolist(), v_analytical.flatten().tolist(), np.zeros_like(u_analytical).flatten().tolist())))
+                combined_stress_true = tuple(np.vstack((sigma_xx_analytical.flatten().tolist(), sigma_yy_analytical.flatten().tolist(), sigma_xy_analytical.flatten().tolist())))
+                
+                error_disp_x = abs(np.array(u_pred) - u_analytical.flatten()).tolist()
+                error_disp_y = abs(np.array(v_pred) - v_analytical.flatten()).tolist()
+                combined_error_disp = tuple(np.vstack((error_disp_x, error_disp_y, np.zeros_like(u_analytical).flatten().tolist())))
+
+                error_stress_x = abs(np.array(sigma_xx_pred) - sigma_xx_analytical.flatten()).tolist()
+                error_stress_y = abs(np.array(sigma_yy_pred) - sigma_yy_analytical.flatten()).tolist()
+                error_stress_xy = abs(np.array(sigma_xy_pred) - sigma_xy_analytical.flatten()).tolist()
+                combined_error_stress = tuple(np.vstack((error_stress_x, error_stress_y,error_stress_xy)))
+
+            x = X[:,0].flatten()
+            y = X[:,1].flatten()
+            z = np.zeros(y.shape)
+
+            if analytical_solution:
+                unstructuredGridToVTK(current_file_path, x, y, z, elements.flatten(), offset, 
+                                    cell_types, pointData = {"pred_displacement" : combined_disp_pred,
+                                                            "pred_stress" : combined_stress_pred,
+                                                            "true_displacement" : combined_disp_true,
+                                                            "true_stress" : combined_stress_true,
+                                                            "error_displacement": combined_error_disp,
+                                                            "error_stress": combined_error_stress,}
+                                                            )
+            else:
+                unstructuredGridToVTK(current_file_path, x, y, z, elements.flatten(), offset, 
+                                    cell_types, pointData = {"pred_displacement" : combined_disp_pred,
+                                                            "pred_stress" : combined_stress_pred}
+                                                            )
+            
+    output_file_pvd =  save_folder_path + "/" + file_name + ".pvd"   
+    create_vtk_collection_file(n=time_steps+1, 
+                                base_filename=file_name,
+                                output_filename=output_file_pvd)
+
+def solutionFieldOnMeshToVtkSpaceTimeBackuP(geom, 
```

Comment:
Is this function still needed (I just got caught up on the name with appended *Backup* 😁 )